packages/core/useFocusWithin/index.ts

```diff
@@ -0,0 +1,32 @@
+import { computed, Ref } from 'vue-demi'
+import { MaybeElementRef, unrefElement } from '../unrefElement'
+import { useActiveElement } from '../useActiveElement'
+import { ConfigurableWindow } from '../_configurable'
+
+export interface FocusWithinOptions extends ConfigurableWindow {
+  /**
+   * The target element to track
+   */
+  target: MaybeElementRef
+}
+
+export interface FocusWithinReturn {
+  /**
+   * True if the element or any of its descendants are focused
+   */
+  focused: Ref<boolean>
+}
+
+/**
+ * Track if focus is contained within the target element
+ *
+ * @see https://vueuse.org/useFocusWithin
+ * @param options
+ */
+export function useFocusWithin(options: FocusWithinOptions): FocusWithinReturn {
```

Comment:
To be consistent with our other API's the first argument should be just the target ref itself. If we have more options they can go in a second options object parameter.

Reply:
Thanks for the clarification. I've gone ahead and made that change. Want me to open a separate PR to update `useFocus` so that it is also consistent? Will be a breaking change, but I would say it's worth it to keep them all in sync.

Reply:
Oh yeah I guess that one slipped by. Feel free to make a PR and we can merge it in the next major version. Thanks!